### ISSUE TITLE
Tool itself can have insufficient contrast

### ISSUE THREAD

**Issue by missmatsuko** [Pushedskydiver/Colour-Contrast-Checker#19]
Since the background color changes based on the colors being compared, the tool itself can fail WCAG contrast requirements.

When I compared [`#C4C4C4` and `#FFFFFF`](https://colourcontrast.cc/c4c4c4/ffffff), the page heading ("colour contrast checker) and HSL sliders fail contrast requirements:
<img width="1397" alt="Screenshot of #C4C4C4 and #FFFFFF comparison in the browser extension." src="https://user-images.githubusercontent.com/17884198/103444606-a4a5ef00-4c1e-11eb-96b7-e677550bcf28.png">

In this example, the background colour is `#C4C4C4` (contrast of 1.74 to background), the heading color is `#FFFFFF`, and the slider bars are `#939393` (contrast of 1.76 to background).

I would recommend not changing the whole tool's background colour, and having a preview swatch instead.

**Comment by bryanjonker-illinois** [comment]
Second this. This is a great tool but I can't read the copy when the accessibility is poor.
